packages/core/stories/button.doc.stories.mdx

```diff
@@ -30,6 +30,8 @@ A Button executes an action when clicked. Button labels should use only one or t
 - **Primary** is the most commonly-used button.
 - **Secondary** performs an action less likely to be triggered.
 
+If the use case requires the user to navigate to a new page or window rather than trigger a function, use a `Link` component instead. The exception to this behaviour is if the navigation is the core action on the page. In this case, a `CTA Button` should be used to trigger the navigation.
```

Comment:
This would be best placed under the suggested 'Do not use when...' heading from the template, but that requires a 'Use when...' heading which we don't have here.

Reply:
@sion-docs i was wondering what to do in this case, given the description of the button is kind of the "Use when" case